---
layout: paper
title: "Validation of a Genotype-Independent Hepatitis C Virus Near-Whole Genome Sequencing Assay."
image: https://www.ncbi.nlm.nih.gov/pmc/articles/instance/8473162/bin/viruses-13-01721-g003.jpg
authors: Hope R Lapointe, Weiyan Dong, Winnie W Y Dong, Don Kirkby, Conan Woods, Art F Y Poon, Anita Y M Howe, P Richard Harrigan, Chanson J Brumme
ref: Lapointe et al. 2021. Viruses.
journal: "Viruses <b>13</b>,  (2021)"
doi: 10.3390/v13091721
abbrev: "Viruses <b>13</b>,  (2021)"
pub_year: 2021
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="34578305" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

Despite the effectiveness of direct-acting antiviral agents in treating hepatitis C virus (HCV), cases of treatment failure have been associated with the emergence of resistance-associated substitutions. To better guide clinical decision-making, we developed and validated a near-whole-genome HCV genotype-independent next-generation sequencing strategy. HCV genotype 1-6 samples from direct-acting antiviral agent treatment-naÃ¯ve and -treated HCV-infected individuals were included. Viral RNA was extracted using a NucliSens easyMAG and amplified using nested reverse transcription-polymerase chain reaction. Libraries were prepared using Nextera XT and sequenced on the Illumina MiSeq sequencing platform. Data were processed by an in-house pipeline (MiCall). Nucleotide consensus sequences were aligned to reference strain sequences for resistance-associated substitution identification and compared to NS3, NS5a, and NS5b sequence data obtained from a validated in-house assay optimized for HCV genotype 1. Sequencing success rates (defined as achieving >100-fold read coverage) approaching 90% were observed for most genotypes in samples with a viral load >5 log10 IU/mL. This genotype-independent sequencing method resulted in >99.8% nucleotide concordance with the genotype 1-optimized method, and 100% agreement in genotype assignment with paired line probe assay-based genotypes. The assay demonstrated high intra-run repeatability and inter-run reproducibility at detecting substitutions above 2% prevalence. This study highlights the performance of a freely available laboratory and bioinformatic approach for reliable HCV genotyping and resistance-associated substitution detection regardless of genotype.

