---
layout: paper
title: "Evaluation of single-cell RNA-seq clustering algorithms on cancer tumor datasets."
image: /assets/images/papers/36420149.png
authors: Alaina Mahalanabis, Andrei L Turinsky, Mia HusiÄ‡, Erik Christensen, Ping Luo, Alaine Naidas, Michael Brudno, Trevor Pugh, Arun K Ramani, Parisa Shooshtari
ref: Mahalanabis et al. 2022. Comput Struct Biotechnol J.
journal: "Computational and structural biotechnology journal <b>20</b>, 6375-6387 (2022)"
doi: 10.1016/j.csbj.2022.10.029
abbrev: "Comput Struct Biotechnol J <b>20</b>, 6375-6387 (2022)"
pub_year: 2022
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="36420149" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

Tumors are complex biological entities that comprise cell types of different origins, with different mutational profiles and different patterns of transcriptional dysregulation. The exploration of data related to cancer biology requires careful analytical methods to reflect the heterogeneity of cell populations in cancer samples. Single-cell techniques are now able to capture the transcriptional profiles of individual cells. However, the complexity of RNA-seq data, especially in cancer samples, makes it challenging to cluster single-cell profiles into groups that reflect the underlying cell types. We have developed a framework for a systematic examination of single-cell RNA-seq clustering algorithms for cancer data, which uses a range of well-established metrics to generate a unified quality score and algorithm ranking. To demonstrate this framework, we examined clustering performance of 15 different single-cell RNA-seq clustering algorithms on eight different cancer datasets. Our results suggest that the single-cell RNA-seq clustering algorithms fall into distinct groups by performance, with the highest clustering quality on non-malignant cells achieved by three algorithms: Seurat, bigSCale and Cell Ranger. However, for malignant cells, two additional algorithms often reach a better performance, namely Monocle and SC3. Their ability to detect known rare cell types was also among the best, along with Seurat. Our approach and results can be used by a broad audience of practitioners who analyze single-cell transcriptomic data in cancer research.

