---
layout: paper
title: "HexSE: Simulating evolution in overlapping reading frames."
image: https://www.ncbi.nlm.nih.gov/pmc/articles/instance/9949996/bin/vead009f2.jpg
authors: Laura Muñoz-Baena, Kaitlyn E Wade, Art F Y Poon
ref: Muñoz-Baena et al. 2023. Virus Evol.
journal: "Virus evolution <b>9</b>, vead009 (2023)"
doi: 10.1093/ve/vead009
abbrev: "Virus Evol <b>9</b>, vead009 (2023)"
pub_year: 2023
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="36846827" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

Gene overlap occurs when two or more genes are encoded by the same nucleotides. This phenomenon is found in all taxonomic domains, but is particularly common in viruses, where it may provide a mechanism to increase the information content of compact genomes. The presence of overlapping reading frames (OvRFs) can skew estimates of selection based on the rates of non-synonymous and synonymous substitutions, since a substitution that is synonymous in one reading frame may be non-synonymous in another and vice versa. To understand the impact of OvRFs on molecular evolution, we implemented a versatile simulation model of nucleotide sequence evolution along a phylogeny with any distribution of open reading frames in linear or circular genomes. We use a custom data structure to track the substitution rates at every nucleotide site, which is determined by the stationary nucleotide frequencies, transition bias and the distribution of selection biases (dN/dS) in the respective reading frames. Our simulation model is implemented in the Python scripting language. All source code is released under the GNU General Public License version 3 and are available at https://github.com/PoonLab/HexSE.

