---
layout: paper
title: "Evaluation of single-cell RNAseq labelling algorithms using cancer datasets."
image: /assets/images/papers/36585784.png
authors: Erik Christensen, Ping Luo, Andrei Turinsky, Mia HusiÄ‡, Alaina Mahalanabis, Alaine Naidas, Juan Javier Diaz-Mejia, Michael Brudno, Trevor Pugh, Arun Ramani, Parisa Shooshtari
ref: Christensen et al. 2023. Brief Bioinform.
journal: "Briefings in bioinformatics <b>24</b>,  (2023)"
doi: 10.1093/bib/bbac561
abbrev: "Brief Bioinform <b>24</b>,  (2023)"
pub_year: 2023
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="36585784" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

Single-cell RNA sequencing (scRNA-seq) clustering and labelling methods are used to determine precise cellular composition of tissue samples. Automated labelling methods rely on either unsupervised, cluster-based approaches or supervised, cell-based approaches to identify cell types. The high complexity of cancer poses a unique challenge, as tumor microenvironments are often composed of diverse cell subpopulations with unique functional effects that may lead to disease progression, metastasis and treatment resistance. Here, we assess 17 cell-based and 9 cluster-based scRNA-seq labelling algorithms using 8 cancer datasets, providing a comprehensive large-scale assessment of such methods in a cancer-specific context. Using several performance metrics, we show that cell-based methods generally achieved higher performance and were faster compared to cluster-based methods. Cluster-based methods more successfully labelled non-malignant cell types, likely because of a lack of gene signatures for relevant malignant cell subpopulations. Larger cell numbers present in some cell types in training data positively impacted prediction scores for cell-based methods. Finally, we examined which methods performed favorably when trained and tested on separate patient cohorts in scenarios similar to clinical applications, and which were able to accurately label particularly small or under-represented cell populations in the given datasets. We conclude that scPred and SVM show the best overall performances with cancer-specific data and provide further suggestions for algorithm selection. Our analysis pipeline for assessing the performance of cell type labelling algorithms is available in https://github.com/shooshtarilab/scRNAseq-Automated-Cell-Type-Labelling.

